src/beautifiers/cljfmt/index.coffee

```diff
@@ -8,12 +8,13 @@ module.exports = class Cljfmt extends Beautifier
   link: "https://github.com/snoe/node-cljfmt"
 
   options: {
-    Clojure: false
+    Clojure:
+      executable_path: true
   }
 
   beautify: (text, language, options) ->
     formatPath = path.resolve(__dirname, "fmt.edn")
-    cljfmt = path.resolve(__dirname, "..", "..", "..", "node_modules/.bin/cljfmt")
+    cljfmt = options.executable_path
```

Comment:
Using the new Executables feature is preferred.

For example:
- OCaml Format
  - https://github.com/Glavin001/atom-beautify/blob/80280bb70c4a10deb3a78d17f7140e81f7486a1e/src/beautifiers/ocamlformat.coffee#L11-L32
- Terraform Fmt
  - https://github.com/Glavin001/atom-beautify/blob/bb9428a4fa483fa6ffe958cdcb519b27e00cb770/src/beautifiers/terraformfmt.coffee#L16-L32
- PHP-CS-Fixer:
  - https://github.com/Glavin001/atom-beautify/blob/8b5363f7bf596a8af03fdee5fdb141b8e22389b7/src/beautifiers/php-cs-fixer.coffee#L55-L56
  - https://github.com/Glavin001/atom-beautify/blob/8b5363f7bf596a8af03fdee5fdb141b8e22389b7/src/beautifiers/php-cs-fixer.coffee#L75-L76
  - https://github.com/Glavin001/atom-beautify/blob/8b5363f7bf596a8af03fdee5fdb141b8e22389b7/src/beautifiers/php-cs-fixer.coffee#L115

Maybe something like:

```coffeescript
executables: [
    {
      name: "cljfmt"
      cmd: "cljfmt"
      homepage: "https://github.com/snoe/node-cljfmt"
      installation: "https://github.com/snoe/node-cljfmt#installation"
      optional: true
      version: {
        parse: (text) -> text.match(/Cljfmt (.*)/)[1]
      }
  ]
```

And then the `beautify` method could be:

```coffeescript
cljfmt = @exe('cljfmt')

defaultCljfmtPath = path.resolve(__dirname, "..", "..", "..", "node_modules/.bin/cljfmt")

cljfmtPath = if cljfmt.isInstalled then cljfmt.path() else defaultCljfmtPath

return @tempFile("input", text).then((filePath) =>
      @run(cljfmtPath, [
        filePath,
        "--edn=" + formatPath
      ]).then(=>
        @readFile(filePath)))
```

This provides:
- ✅ Backwards compatibility
- ✅ Uses Executables (standardized, better support)
- ✅ Allows customization of the path via Executable's auto-generated option, `cljfmt_path`

Please let me know if you have any questions or need help implementing!